tests/integration/components/bs-popover-test.js

```diff
@@ -78,6 +79,13 @@ module('Integration | Component | bs-popover', function(hooks) {
     this.insertCSSRule('#wrapper p { margin-top: 200px }');
 
     let expectedArrowPosition = versionDependent(225, 219);
+
+    if (isFirefox()) {
+      // Popover arrow has a different positioning by 4px on Firefox
+      // https://github.com/twbs/bootstrap/issues/29485
+      expectedArrowPosition = versionDependent(225 - 4, 219 - 4);
```

Comment:
```suggestion
      expectedArrowPosition = expectedArrowPosition - 4;
```

This would make it a bit more clear that FF is always 4px off, and makes it less error prone in case the base values (225/219) need to change, e.g. in case of changing the test. Just a very minor suggestion though